code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -960,34 +971,34 @@
 	flags_inv = HIDEJUMPSUIT
 	resistance_flags = NONE
 
-/obj/item/clothing/suit/hooded/wintercoat/christmascoatr
+/obj/item/clothing/suit/hooded/wintercoat/false/christmascoatr
```

Comment:
also you might want to just define a helper for false coats or something and add new paths. 
you're removing existing wintercoats because they're on loadout.
it's not preferable compared to making new paths altogether.